lib/javascript/JavascriptParser.js

```diff
@@ -2657,7 +2657,7 @@ class JavascriptParser extends Parser {
 	 * @param {MetaPropertyNode} metaProperty meta property
 	 */
 	walkMetaProperty(metaProperty) {
-		this.hooks.metaProperty.call(metaProperty);
+		this.hooks.metaProperty.for(getRootName(metaProperty)).call(metaProperty);
```

Comment:
Do we even need a separate hooks for that or could we just use the `expression` hook?